cart/cart.py

```diff
@@ -604,7 +604,7 @@ def main():
                             backup_name = backup_name[:-5]
                         else:
                             backup_name += ".uncart"
-                        output_file = cur_metadata.get("name", backup_name)
+                        output_file = cur_metadata.get("name", backup_name).lstrip("/")
```

Comment:
How would this play out on other OS like Windows? Maybe we should strip/replace any path separators?